browser/ui/views/frame/brave_browser_view.h

```diff
@@ -19,6 +19,7 @@
 #include "brave/browser/ui/tabs/brave_tab_strip_model.h"
 #include "brave/browser/ui/tabs/split_view_browser_data.h"
 #include "brave/browser/ui/tabs/split_view_browser_data_observer.h"
+#include "brave/browser/ui/views/split_view/split_view_location_bar.h"
```

Comment:
nit: can we forward declare this? I'm always a bit hazy about if you can do it with `raw_ptr`

Reply:
Oh sure. My autocomplete filled it.

Reply:
Mine does it all the time! It's super helpful mostly :laughing: